lib/ansible/modules/source_control/git.py

```diff
@@ -42,10 +42,10 @@
     dest:
         required: true
         description:
-            - Absolute path of where the repository should be checked out to.
-              This parameter is required, unless C(clone) is set to C(no)
-              This change was made in version 1.8.3. Prior to this version,
-              the C(dest) parameter was always required.
```

Comment:
This would be a good occasion to remove the comment on versions before 1.8.3. Nobody reading the docs nowadays will care about that any more.